doc/getting-started/getting-started.rst

```diff
@@ -741,20 +598,19 @@ The ACRN hypervisor boots the Ubuntu Service VM automatically.
 #. On the target, log in to the Service VM. (It will look like a normal Ubuntu
    session.)
 
-#. Verify that the hypervisor is running by checking ``dmesg`` in
-   the Service VM:
+#. Verify that the hypervisor is running by checking ``dmesg`` in the Service
+   VM:
 
    .. code-block:: bash
 
-      dmesg | grep ACRN
+      dmesg | grep -i acrn
 
    You should see "Hypervisor detected: ACRN" in the output. Example output of a
    successful installation (yours may look slightly different):
 
    .. code-block:: console
 
       [  0.000000] Hypervisor detected: ACRN
```

Comment:
I see this output:
```
[    0.000000] Linux version 5.10.90-acrn-service-vm (dbkinder@dbkinder-ThinkPad-X220) (gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0, GNU ld (GNU Binutils for Ubuntu) 2.34) #2 SMP PREEMPT Mon Feb 28 16:54:33 PST 2022
[    0.000000] Hypervisor detected: ACRN
[    1.830455] usb usb1: Manufacturer: Linux 5.10.90-acrn-service-vm xhci-hcd
[    1.830728] usb usb2: Manufacturer: Linux 5.10.90-acrn-service-vm xhci-hcd
[    1.832545] usb usb3: Manufacturer: Linux 5.10.90-acrn-service-vm xhci-hcd
[    1.834425] usb usb4: Manufacturer: Linux 5.10.90-acrn-service-vm xhci-hcd
[    1.948464] ACRNTrace: Initialized acrn trace module with 4 cpu
[    1.948464] ACRN HVLog: no fixed memory reserve for hvlog.
[    2.799179] systemd[1]: Detected virtualization acrn.
```

Maybe we should grep for "hypervisor" instead of "acrn", in which case only this one line shows up:
```
[    0.000000] Hypervisor detected: ACRN
```